students/axel_nowaczyk/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,87 @@
+def show_help():
+    print("HELP")
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def perform_addition():
+    print("ADDING")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 + add_var_2)
+
+
+def perform_subtraction():
+    print("SUBTRACT")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 - add_var_2)
+
+
+def perform_multiplication():
+    print("MULTIPLY")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 * add_var_2)
+
+
+def perform_division():
+    print("DIVIDE")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 / add_var_2)
+
+
+def perform_power():
+    print("POWER")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 ** add_var_2)
+
+
+def start_calculator():
```

Comment:
You can use sth like:
options = {
   'a': parform_addition
   ....
}
 and later 
options[option]()

but your solution is good too